CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+### Change Log
+
+All notable changes to this project will be documented in this file.
+
+### Sprint 66 ending 2017-08-07
+- Force array context for VMs vnets and images [(#13)](https://github.com/ManageIQ/manageiq-providers-scvmm/pull/13)
```

Comment:
Awesome,...I'll update with that info and do the same for the others I'm working on.